viaq-jenkins/README.md

```diff
@@ -0,0 +1,37 @@
+com.redhat.viaq-ci template
+=================================
+
+The template file com.redhat.viaq.template.json is automatically generated. Please *do not edit* the file directly.
+
+In order to edit the template please modify fields.yml and skeleton.json files.
+
+To rebuild the template, run:
+> make clean && make
+
+For details about the mapping please see [ElasticSearch reference](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-templates.html)
+
+skeleton.json
+-------------
+This file contains the skeleton of the template without the mappings section.
+
+by sections:  
+`aliases`: Aliases for the indices produces from the template. Aliases will be automatically generated along with the indices in future.  
+`mappings`: mappings section  
+  `_default_`: default mapping skeleton. It is the only mapping present in the skeleton. Other mappings from [fields.yml](fields.yml) will copy this skeleton.  
+  `date_detection`: we force no date detection in the unknown fields.  
+  `dynamic_templates`: describes the mapping for autocreated fields.  
+  `properties`: empty section that is populated with the content from [fields.yml](fields.yml)  
+`order`: order of the template. lower order templates are applied first.  
+`settings`: various settings  
+`template`: indices that will be matched by this template  
+
+fields.yml
+----------
+This is the file that contains all the mappings information.  
+`version`: version of the mapping  
```

Comment:
enhance upon:
this is intended to be used by a specific normalizer configured to use this particular version